keanu-project/src/main/java/io/improbable/keanu/algorithms/mcmc/SamplingAlgorithm.java

```diff
@@ -17,9 +17,10 @@
      * Takes a sample with the algorithm and saves it in the supplied map (creating a new entry in the list if the
      * Vertex already exists).
      *
-     * @param samples map to store sampled vertex values
+     * @param samples  map to store sampled vertex values
+     * @param logProbs list of log probabilities for each sample
```

Comment:
Might be worth documenting what the "LogProb" is here and how it lines up with the samples Map (ie presumably length logProbs == length of each of the values from the samples Map